### PR TITLE
Create Router 

### PR DESCRIPTION
**Pull request by PierreGoe**

### CODE REVIEW

**Review by HeloiseFerla — DISMISSED** [review]
To be modified when we will develop the cocktailDetails

File: src/App.jsx [hunk 1]
```diff
@@ -1,14 +1,26 @@
...
 
 function App() {
   return (
-    <>
-      <Header displaySearch />
-      <CocktailView />
-      <Footer />
-    </>
+    <Router>
+      {' '}
```
Comment (LoicBrassart) [inline src/App.jsx]: Useless

File: src/App.jsx [hunk 2]
```diff
@@ -1,14 +1,26 @@
...
+      <>
```
Comment (LoicBrassart) [inline src/App.jsx]: Useless Fragment (artifact from before-router ?)

File: src/index.jsx [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './reset.css';
+
```
Comment (LoicBrassart) [inline src/index.jsx:4]: ... Why ?

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Minor Evils to fix, see comments

File: src/App.jsx (hunk 1 above)
Comment (Morel-Victor) [reply src/App.jsx]: Useless line resolved

File: src/App.jsx (hunk 2 above)
Comment (Morel-Victor) [reply src/App.jsx]: Useless fragment removed

File: src/index.jsx (hunk 3 above)
Comment (Morel-Victor) [reply src/index.jsx:4]: Useless white space removed

**Review by HeloiseFerla — DISMISSED** [review]
ok

**Review by LoicBrassart — DISMISSED** [review]
No more Evils ! \o/

**Review by HeloiseFerla — APPROVED** [review]
ok

**Review by Hacene010 — APPROVED** [review]
It's ok for me ;)

**Pull request merged**